### ISSUE TITLE
create_installer_mac: notarize stapling failing rc=73

### ISSUE THREAD

**Issue by andrew-m-leonard** [AdoptOpenJDK/openjdk-build#2208]
https://ci.adoptopenjdk.net/job/build-scripts/job/release/job/create_installer_mac/5235/console
```
Size of data is 5617
JSON Response is: {
    records =     (
                {
            created =             {
                deviceID = 2;
                timestamp = 1604433764989;
                userRecordName = "_d28c74d190a3782e89496b0a13437fef";
            };
            deleted = 0;
            fields =             {
                signedTicket =                 {
                    type = BYTES;
                    value = "czhjaAEAAADvBQAAYQgAADCCBeswggL9MIICpKADAgECAghTHMPRWnzdaDAKBggqhkjOPQQDAjByMSYwJAYDVQQDDB1BcHBsZSBTeXN0ZW0gSW50ZWdyYXRpb24gQ0EgNDEmMCQGA1UECwwdQXBwbGUgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkxEzARBgNVBAoMCkFwcGxlIEluYy4xCzAJBgNVBAYTAlVTMB4XDTIwMDUwNTE2NTcwMloXDTIxMDYwNDE2NTcwMlowRDEgMB4GA1UEAwwXU29mdHdhcmUgVGlja2V0IFNpZ25pbmcxEzARBgNVBAoMCkFwcGxlIEluYy4xCzAJBgNVBAYTAlVTMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEDZ54cNjcv0P2fiHOO52zhKBFldpgd5aCoCnXpnVpezwePurTq8bK1ynB7s1Xk9JnXHWExghzTKxlr4U0hN1BeKOCAVAwggFMMAwGA1UdEwEB/wQCMAAwHwYDVR0jBBgwFoAUeke6OIoVJEgiRs2+jxokezQDKmkwQQYIKwYBBQUHAQEENTAzMDEGCCsGAQUFBzABhiVodHRwOi8vb2NzcC5hcHBsZS5jb20vb2NzcDAzLWFzaWNhNDAyMIGWBgNVHSAEgY4wgYswgYgGCSqGSIb3Y2QFATB7MHkGCCsGAQUFBwICMG0Ma1RoaXMgY2VydGlmaWNhdGUgaXMgdG8gYmUgdXNlZCBleGNsdXNpdmVseSBmb3IgZnVuY3Rpb25zIGludGVybmFsIHRvIEFwcGxlIFByb2R1Y3RzIGFuZC9vciBBcHBsZSBwcm9jZXNzZXMuMB0GA1UdDgQWBBTM0eQEHQPM/nSI32QVbNrUFwRfkTAOBgNVHQ8BAf8EBAMCB4AwEAYKKoZIhvdjZAYBHgQCBQAwCgYIKoZIzj0EAwIDRwAwRAIgbO2fArMDD8WWpuv2ty72ibhg2P0ECk5HJFHalVXt+fgCIHaDQAwlBJsJm0x+LY7j+1QtB0DJI1RTN89eDa6gHzs4MIIC5jCCAm2gAwIBAgIIMw3u+L9MaC4wCgYIKoZIzj0EAwMwZzEbMBkGA1UEAwwSQXBwbGUgUm9vdCBDQSAtIEczMSYwJAYDVQQLDB1BcHBsZSBDZXJ0aWZpY2F0aW9uIEF1dGhvcml0eTETMBEGA1UECgwKQXBwbGUgSW5jLjELMAkGA1UEBhMCVVMwHhcNMTcwMjIyMjIyMzIyWhcNMzIwMjE4MDAwMDAwWjByMSYwJAYDVQQDDB1BcHBsZSBTeXN0ZW0gSW50ZWdyYXRpb24gQ0EgNDEmMCQGA1UECwwdQXBwbGUgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkxEzARBgNVBAoMCkFwcGxlIEluYy4xCzAJBgNVBAYTAlVTMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEBmukVm99nyfRzjaOkhtWzVQ2ZErJlGiZ+skgfuL1WA/c4mrrGUcvLu87pAG0ARNEfFomraCcKSWK5eYGb098WqOB9zCB9DAPBgNVHRMBAf8EBTADAQH/MB8GA1UdIwQYMBaAFLuw3qFYM4iapIqZ3r6966/ayySrMEYGCCsGAQUFBwEBBDowODA2BggrBgEFBQcwAYYqaHR0cDovL29jc3AuYXBwbGUuY29tL29jc3AwMy1hcHBsZXJvb3RjYWczMDcGA1UdHwQwMC4wLKAqoCiGJmh0dHA6Ly9jcmwuYXBwbGUuY29tL2FwcGxlcm9vdGNhZzMuY3JsMB0GA1UdDgQWBBR6R7o4ihUkSCJGzb6PGiR7NAMqaTAOBgNVHQ8BAf8EBAMCAQYwEAYKKoZIhvdjZAYCEQQCBQAwCgYIKoZIzj0EAwMDZwAwZAIwFQypjsavlmlrp5/dXNQDWWyuUtRgxot24LfFovEJfOCa42ux43wxxCd6p46J/at3AjBhMyDxKf/5hzKuKchkPXZ7UaTSAi92vmAikfHVOnXctOLGKpb+xgncSk/VJPD8yrJnOHRrAgAUAGUAAAAAAAAAZLehXwAAAAABrJHoMX42UPzFMm8yZTBfOdfyo/QCb64dE1pVKQAJtXyPhROea2cuLawCU20/4nshmADu9nrz+aWEH/If8fgCFJH1hP9jK4ocJmgG4Q2TwffCAtgCfFeLC8x48uvY/Z+uDRJViLJ3PEsCZlTpHpZc2RyBl7fSA18HtEPkYK8CifaK1WJLkBJjcKv8YMKss3vsccIC8Y6kdF95ilx2L3nG/TMZr9xJ30EC7ov32mxx//RgToMmTlh4CYnqe1MCv/HYlZ0eV6mPqMwqK+vBpsUaocUCuGL7L9FLNS8P8FYbN4jTZcjpyKkC173HDbGw3C1g/RCtT2cDruP9KI0CHOivUHI+K9KnjDGYINQK0+eX+pYCrHpvw9LgaqP9RBmV7PQdk7NQGD0COWr0gzBfPCWmRr3j2UgczE8SkXUCohBYmpd5Qsrvjj1CrD7rMy0zhqoCu2pHphk6flNNVKYVgjk1Tz3SZa8C2lIqgAFRTQdsrRVdQ1/244CJ2j0CGdotSQAyphD/kdSo4RMuvYh2fi0CjMqJlIepGhsLUS7qOBRXT550VsECvFDhN7za8P8sfa0Fa5/tnPxLDLwCxAU87UxI/FVxVpRfZ5N6uGXu0VECDtpxhYuTLlMQzMm0osbb1Zxw76UCTFlpXywB/5e4WPanIUrMy0STBBwCpMhZ4J5yPpWjImnd0SkjIlArCXcCx/LnfbSqss2iU51yeFWjC6AyiF8CGmW2MNVL2lPG4qzNjf7Tbs8dHWQCGMB/f6BlP4vB6y3BhSEwflrfsnYCf/le/zehhsJ7XFPIH6mF/PwrdtYCOTyXr548yV5nqdoyan2k68qNUFoC0thBNg/nMosHdQQ9ybn5AXXm3/0CNcOtCFFcmj0tbGIVwtlZRViWTZcCEg/TvqF/nEcv11cxHc0NtwcH9+sCLBcvQ75jN9qDPKc2kwy4xvk9D78CUUwK7nA2DZG1KLcaXydyNl1wlXsCBvqUTy44gy4IAtKpace7N+8afgQCRZFQWVxTKMoBzcGZxrudUp82N+ACyHhPRwjNkojlrjYb116irjUmtVAC/cTwGPZriVpaocguhqgu5Wa9wW0CPd3CYpzThK5S+xpGKriFSfAUSRQCVP/VNDdB1m4BkuE8r8t2CuIA8b4CwkhbDJ9UGy/w0+oPa42oUbuRD/4CzLpYSGIS2zrfPRg2RVsqsqjSru0C2vQlPyP/8yel29OdmOPYEMvVtX4CGqcyeKCwIlg7Jmn0HfAEllZ7xIMCVC1mr2sER8sGgsXyf09jhrNI7PECR2xiHeaGBVxg/d21+vciQuwJKtcCjcK4u+gGEx1L1HWzFVNGlphJpQwCqKc7Q4BhxracqYmGbvvCicUXp+gCsFblxqpVOBQvp5FRSkSCBISZgHEC4s5SeTDS64VRncTF262lgKCzXy0CA4cYYTnbhw+2i7Rxlh5Fu1neBUkC9n+4AucmZaPjL/KDasNxho5bh7kCRvWg9LKalSr5DkAwSBfc7zKCpZMCQT+BT6MOm3PlaW5Gl0bLLuAyZ6QC9J6TqlB9gTTdNUNdl26AtKpdojsCu5fHEDy65Z6PMWVX8KRNgjwM40wCUf4QGBcy11yfe8fRAXv1CAtANtYCQniZLKIIdh9YQ+orafXC03WBwO4CHMp9O56FboxPpwEO4MOf/LqG8kIC714ZVWrONQ3JRI035OmylCw97BACzlzQHDXyLMfItMfoxl2RbnT8bBQCjlS2wEynxOV5SiWS4MgZob3B5BYCTfdGiV/ykDgudFNceq2JkupxIHkCNYRJiT7mTU+WPhjN+82ZxgiT4M8CCq5rUm2Z3sVIiC6DN0ceGSsW3vACmffIk9+I5KjKO5zl3qqhKW6ob1QCPSbucVc1Is8nZV+/6Pyr0W0pCfsCAFTMkoOmM08rr0wrhK41qouPEJICxrkV9Ct0g21BcMIsIdMLl6ptJkICO1upINh6aALVrvRajSV/W4pYHeECNRt3gr+p2dtnHVK8Yk/01/B+e+wClrVZ04y33RcOQVc9VyR0/PVdzycCrM4ng7KwVpzbaaE3xOO439Dq82MClf1+Tk91CEH3Pxs7gx3SsNKK9TgCEZE7ohrX+kkKGXg4udJNRy2dISUC0UOsK4YRiPPrxAWAw1zt3dA0uVkCF6sHVny9HdJY7R6Z/OCGA83zrb4CJ35nSLzf2I2puPe7XrNs7LZes4YC8tcqJdKPC2xAN/AQOZ3CVcnrKAUCeyc3FNHesGwI3QgNbmmlYuVNGxACcSSrKmkIx7xZ/VdrxNRh8afEr4MCMRDqyxxE31sUhqCAcxkMLsbq4XMCbEfNAvgmPdKfBBpg8FcmtzrceRUCU+lTiwYWIyYmmFZZHNczy/4ql8cCuqXpt0Oa2magst1z175hHDtYuTwCts27NVQfRuavPYb5H+ZcD+W24e8CiAZHSSOAxwOjjHmP4sq5FrczucoCrkS75WS7X8j9Yy8fmsqWTnJBkhsCg8qrZyZiDW3qCn+Z06TKfPTZNIsCS3kQBIMChnhbCYFXfHb3sUKOsDECJ158jAr8mUZK6oYK1vVV5cZ6mgACLVvWNFq0zlUNYICCz2Ixrj+B0ZIC8sNjdQbtpf3vNgeV+HAEdK8AEbgC/ojmrbo4w2/axb8tTYBSfxAEfG8CPtugzREcLCMtXOtTc5NRL5e9L+4CTM39hqwqoDjvhwUWtedDKCEB7xACNqLT91NboMFwwxflHS/rfsUVQ8QCmGVRGTnXunTZQ+fsC1ieBwkvCo0CMznchF0fhEw69CCK2JPRbXL/6t8CmknD8d5tbKFFg3FylAg8giPCBtAwRgIhAOYIVInRM5EN5P3v8aaSO37YI6VDX/OKVxHkmmlLLir/AiEA000kd9bqCQTHXk/xlditG37vuTnHJqfc+gFupbM35Ps=";
                };
            };
            modified =             {
                deviceID = 2;
                timestamp = 1604433764989;
                userRecordName = "_d28c74d190a3782e89496b0a13437fef";
            };
            pluginFields =             {
            };
            recordChangeTag = kh2efs6x;
            recordName = "2/1/ac91e8317e3650fcc5326f3265305f39d7f2a3f4";
            recordType = DeveloperIDTicket;
        }
    );
}
Downloaded ticket has been stored at file:///var/folders/52/s8sjmz9n2_s_m4zh89f5fdfr0000gp/T/c7d381d9-7361-4a59-b89d-a690ba807b13.ticket.
Attempting to attach a new ticket to OpenJDK8U-jdk_x64_mac_openj9_2020-11-03-17-06.pkg. Let's see how that works out.
Processing: /Users/jenkins/workspace/build-scripts/release/create_installer_mac/workspace/target/OpenJDK8U-jdk_x64_mac_openj9_2020-11-03-17-06.pkg
Properties are {
    NSURLIsDirectoryKey = 0;
    NSURLIsPackageKey = 0;
    NSURLIsSymbolicLinkKey = 0;
    NSURLLocalizedTypeDescriptionKey = "Installer package";
    NSURLTypeIdentifierKey = "com.apple.installer-package-archive";
    "_NSURLIsApplicationKey" = 0;
}
We do not know how to deal with trailer version 50357. Exepected 1
Although we wrote the ticket, the written data did not validate. Please restore /Users/jenkins/workspace/build-scripts/release/create_installer_mac/workspace/target/OpenJDK8U-jdk_x64_mac_openj9_2020-11-03-17-06.pkg from backup to try again.
The staple and validate action failed! Error 73.

    at /Users/jenkins/workspace/build-scripts/release/create_installer_mac/pkgbuild/notarize/node_modules/electron-notarize/lib/index.js:135:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/jenkins/workspace/build-scripts/release/create_installer_mac/pkgbuild/notarize/node_modules/electron-notarize/lib/index.js:4:58)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
Build step 'Execute shell' marked build as failure
Archiving artifacts
[Checks API] No suitable checks publisher found.
Finished: FAILURE
```

**Comment by gdams** [comment]
fixed, I had to install Xcode 10.3 on the new machines (to match old ones) instead of Xcode 10.1

**Comment by gdams** [comment]
It also seems that I needed to add this fix: https://github.com/AdoptOpenJDK/openjdk-installer/commit/fe8c79d25551401db3cb06b3c7a72f013a56051c
